src/main/java/com/replicon/bookTimeOff/ViewTimeOffRequestsPage.java

```diff
@@ -13,7 +13,9 @@ public class ViewTimeOffRequestsPage extends WebPage {
     protected WebElement allTimeOffBookings;
     @FindBy(css = "a[id='All'] > span:nth-child(2)")
     protected WebElement allRequests;
```

Comment:
the webelement name "all request" is vague; the name could have been "total number of all request"